cor_surrogates/code/.Rprofile

```diff
@@ -0,0 +1 @@
+source("renv/activate.R")
```

Comment:
And remove this file :)

Reply:
Sure thing done; I didn't realize how these sneaked in.

Reply:
Yes, I've started to be a bit more careful with my `git add` commands. It's easy to sneak in `renv`-associated files.

Reply:
Thanks David; will be careful as well from my side. Back to the python thing. Will update you in a bit.